src/leaflike/bookmarks/core.clj

```diff
@@ -0,0 +1,46 @@
+(ns leaflike.bookmarks.core
+  (:require [leaflike.bookmarks.validator :refer [valid-bookmark?
+                                                  id?]]
+            [leaflike.bookmarks.db :as bm-db]
+            [leaflike.user.db :as user-db]
+            [leaflike.user.auth :refer user-session]
+            [leaflike.utils :as utils]))
+
+(defn- get-user
+  []
+  (when-not (nil? @user-session)
+    (let [identifier (get @user-session :identity)]
+      (first (user-db/get-member-auth-data identifier :id)))))
+
+(defn create
+  [request]
+  (let [body    (-> request :body)
```

Comment:
For a single fn call you can use the (fn args) syntax and reserve threaded macros when there's more than one fn call involved